recipes/asv_bench_memray/meta.yaml

```diff
@@ -0,0 +1,44 @@
+{% set name = "asv_bench_memray" %}
+{% set version = "0.1.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/asv_bench_memray-{{ version }}.tar.gz
+  sha256: 654a5bb989e15d2a6b46d85349b492f83605ae1e89d95f76f8f5943d5723cb8f
+
+build:
+  skip: true  # [win]
```

Comment:
Looks like this on could be noarch.

It is OK for noarch package to fails on Windows with a missing dependency. We prefer noarch b/c it is faster to build and the package will be ready when the missing dependency is available on Windows.

Reply:
Makes sense, updated, sorry for the long delay!!